src/java/org/apache/cassandra/io/util/RandomAccessReader.java

```diff
@@ -43,6 +42,7 @@ public class RandomAccessReader extends RebufferingInputStream implements FileDa
     final Rebufferer rebufferer;
     private BufferHolder bufferHolder = Rebufferer.EMPTY;
     private final ByteOrder order;
+    ByteBuffer temporaryBuffer;
```

Comment:
Nit: is it worth making it `private`?

Reply:
yes, initially I wanted to test it in unit tests, but it is overkill

moved to `private`